src/Processors/AugmentTags.php

```diff
@@ -15,6 +15,22 @@
  */
 class AugmentTags implements ProcessorInterface
 {
+
+    /** @var array<string>  */
+    protected $unusedTagsToKeepWhitelist = [];
```

Comment:
```suggestion
    protected $unusedTagsToKeepWhitelist;
```
I think the other processors only intialize in the constructor and that should be enough.

Reply:
based it off CleanUnusedComponents, makes sense to only do it in one spot though.